packages/flutter_tools/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   archive: 2.0.13
   args: 1.6.0
   dds: 1.3.1
-  dwds: 5.1.0
+  dwds: 3.1.3
```

Comment:
@liyuqian I think this is the trouble. Somehow `flutter update-packages --force-upgrade` is downgrading this package. I posted a question in discord.